content/source/docs/enterprise/api/policy-sets.html.md

```diff
@@ -148,16 +223,17 @@ Status  | Response                                      | Reason
 [200][] | [JSON API document][] (`type: "policy-sets"`) | Request was successful
 [404][] | [JSON API error object][]                     | Organization not found, or user unauthorized to perform action
 
-
 ### Query Parameters
 
 This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
 
-Parameter      | Description
----------------|------------
-`page[number]` | **Optional.** If omitted, the endpoint will return the first page.
-`page[size]`   | **Optional.** If omitted, the endpoint will return 20 policy sets per page.
-`search[name]` | **Optional.** Allows searching the organization's policy sets by name.
+Parameter           | Description
+--------------------|------------
+`filter[versioned]` | **Optional.** Allows filtering policy sets based on whether they are versioned (VCS or API), or manually updated. Accepts a boolean true/false value. If omitted, all policy sets are returned.
+`include`           | **Optional.** Allows including related resource data. Value must be one of `workspaces`, `policies`, or `current_version`.
```

Comment:
I think it's good you're doc'ing this `include` stuff as we're really bad at it in general, but where we do doc it we usually leave it as a separate section, [like in the Workspaces API](https://www.terraform.io/docs/enterprise/api/workspaces.html#available-related-resources). Up to @nfagerlund really, I just thought I'd point out where we're differing from the consistent norm here.

Reply:
(though if we're documenting the pagination stuff here I see no reason not to also...include....`include`, too) 🤷‍♂

Reply:
I had also noticed that, and thought "this is probably where I'd come looking for this option", so thus I included it here.